scripts/speaker_tasks/pathsfiles_to_manifest.py

```diff
@@ -45,24 +45,38 @@ def read_file(scp):
     return sorted(scp)
 
 
-def main(wav_scp, rttm_scp=None, uem_scp=None, manifest_filepath=None):
+def main(wav_scp, text_scp=None, rttm_scp=None, uem_scp=None, ctm_scp=None, manifest_filepath=None):
     if os.path.exists(manifest_filepath):
         os.remove(manifest_filepath)
```

Comment:
It's too dangerous to just delete the existing file here.

Reply:
> Mostly looks good. Some comments and suggestions. And maybe include multiscale performance somewhere, say readme, on benchmark dataset.

README will be entirely updated with asr_with_diarization PR. so let me add those when we push them

Reply:
this script is not part of this PR (just fixed some small part of the script on @nithinraok's request.). Since this "remove" function also appears in diarization pipeline, let's discuss this convention again to fix all those together.

Reply:
Okay cool